examples/camel-example-spring-boot-amq/readme.adoc

```diff
@@ -0,0 +1,23 @@
+# Camel Example Spring Boot and ActiveMQ
+
+This example shows how to work with a simple Apache Camel application using Spring Boot and Apache ActiveMQ starter
+
+##prerequisites
+A running Apache ActiveMQ instance with openwire transportConnector opened on port 1234
```

Comment:
Sure but the purpose of using another port is to show the problem when overriding the default cf broker_url since localhost:61616 is default. For the example though it would be easier to use default, but to reproduce the cf issue you would have to use another port or host.